model_hub/examples/huggingface/multiple-choice/swag_config.yaml

```diff
@@ -0,0 +1,42 @@
+description: swag_trial
```

Comment:
Generally, for the descriptions, can we have `huggingface` as part of them for representation of the framework like we do for `iris_tf_keras`?